contrib/storage-mongo/src/test/java/org/apache/drill/exec/store/mongo/MongoTestSuit.java

```diff
@@ -128,42 +130,63 @@ private static void setup() throws Exception {
       createDbAndCollections(DATATYPE_DB, DATATYPE_COLLECTION, "_id");
     }
 
-    private static IMongodConfig crateConfigServerConfig(int configServerPort,
-        boolean flag) throws UnknownHostException, IOException {
-      IMongoCmdOptions cmdOptions = new MongoCmdOptionsBuilder().useNoJournal(false).verbose(false)
-          .build();
+    private static IMongodConfig crateConfigServerConfig(int configServerPort) throws UnknownHostException, IOException {
+      IMongoCmdOptions cmdOptions = new MongoCmdOptionsBuilder()
+        .useNoPrealloc(false)
+        .useSmallFiles(false)
+        .useNoJournal(false)
+        .useStorageEngine(STORAGE_ENGINE)
+        .verbose(false)
+        .build();
+
+      Storage replication = new Storage(null, CONFIG_REPLICA_SET, 0);
 
       IMongodConfig mongodConfig = new MongodConfigBuilder()
-          .version(Version.Main.PRODUCTION)
+          .version(Version.Main.V3_4)
           .net(new Net(LOCALHOST, configServerPort, Network.localhostIsIPv6()))
-          .configServer(flag).cmdOptions(cmdOptions).build();
+          .replication(replication)
+          .shardServer(false)
+          .configServer(true).cmdOptions(cmdOptions).build();
```

Comment:
Please move to the new lines `.cmdOptions(cmdOptions).build();`

Reply:
Successfully built Mongo storage plugin on my Mac.  LGTM +1